### PR TITLE
Webpack, Less, file moving

### PR DESCRIPTION
**Pull request by SHBelsky**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +0,0 @@
-{
```
Comment (karunsingh) [inline .vscode/settings.json]: Could we keep these VS Code settings?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
 # Office Hours
-./client contains the development server for the react app, while ./server contains the GraphQL server for the backend
+`./client` contains the development server for the react app, 
+while `./server` contains the GraphQL server for the backend
 
 ## Setup
-Run
+
+1. Install Yarn [here](https://yarnpkg.com/en/docs/install).
```
Comment (karunsingh) [inline README.md]: Why do we need yarn?

Comment (karunsingh) [reply README.md]: (Can we stick to npm?)

**Comment by karunsingh** [comment]
@SHBelsky As far as adding a CSS pre-processor, it looks like create-react-app has some instructions of its own: https://github.com/cornell-dti/office-hours/tree/master/client#adding-a-css-preprocessor-sass-less-etc

**Comment by SHBelsky** [comment]
I'm in Israel at the moment, so I'm away from my computer until Saturday. I'll revisit this when I'm back.

**Pull request closed without merging**